ChartboostMediationAdapterAdColony.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |spec|
   spec.module_name  = 'ChartboostMediationAdapterAdColony'
   spec.source       = { :git => 'https://github.com/ChartBoost/chartboost-mediation-ios-adapter-adcolony.git', :tag => spec.version }
   spec.source_files = 'Source/**/*.{swift}'
+  spec.resource_bundles = { 'ChartboostMediationAdapterAdColony' => ['Source/PrivacyInfo.xcprivacy'] }
```

Comment:
Same idea as in this PR: https://github.com/ChartBoost/chartboost-core-ios-sdk-private/pull/175/files

Reply:
Same as in the other PR, should we put this file in the parent directory?

Reply:
updated now